### DISCUSSION TITLE
Welcome to ecModels

### DISCUSSION THREAD

**Discussion by IVANDOMENZAIN** [SysBioChalmers/ecModels#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * [Ask questions](https://github.com/SysBioChalmers/ecModels/discussions/categories/q-a) about the features of our ecModels or the coding of our pipeline.
  * [Share ideas](https://github.com/SysBioChalmers/ecModels/discussions/categories/ideas) for further development or propose addition of new models to our catalogue.
  * [Engage](https://github.com/SysBioChalmers/ecModels/discussions/categories/general) with other community members.
  * [Share](https://github.com/SysBioChalmers/ecModels/discussions/categories/show-and-tell) your own applications or upgrades of our ecModels with the community.
  * Welcome others. Remember that this is a community we
  build together 💪.

<!--

**Comment by noushineftekhari** [comment]
Hi, I am working with the ecModel, I need to ask a couple of questions: 
I want to use proteomic data from TCGA, but the values are log 2 of relative abundance, I removed the log but the range still is not the same with PaxDb data, I think because they divided with SD as well.
Did you have any experience with other proteomic data from TCGA?
Can help me out, the GECKO is able to work with TCGA format or not?
